src/com/array/CommonElement.java

```diff
@@ -0,0 +1,22 @@
+package com.array;
+
+import java.util.Arrays;
+
+public class CommonElement {
+
+	public static void main(String[] args) {
+		int[] a = {1,2,3,4,5,6};
+		int[] b = {7,8,9,10,11,12};
+		Arrays.sort(a);
```

Comment:
Do not unnecessorily increase the code

Reply:
It is written Because if order changed then Array.sort is required